### DISCUSSION TITLE
dotnet core: debugger is using DLL from cached image?

### DISCUSSION THREAD

**Discussion by cutecycle** [microsoft/vscode-docker#3273] [Q&A]
Right now my container is throwing an error from a line of code that was removed. Is there an additional launch.json flag somewhere to --no-cache for launches, or should "build image" be run every time?

**Comment by karolz-ms** [comment]
@bwateratmsft I thought dotnet debugging was using bind mounts, so it should get the most fresh bits?

**Reply by bwateratmsft** [reply]
Regular "single container" debugging does, but if it's using Compose debugging it might not be the case. @cutecycle are you building your container with a `docker-build` task or with a Compose build?
